src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -83,7 +83,7 @@ fun snykCodeAvailabilityPostfix(): String = when {
     !isSnykCodeAvailable(pluginSettings().customEndpointUrl) -> " (disabled for endpoint)"
     (pluginSettings().sastOnServerEnabled ?: true && pluginSettings().localCodeEngineEnabled ?: true) ->
         " (disabled due to Local Code Engine)"
-    !(pluginSettings().sastOnServerEnabled ?: false) -> " (disabled for organization)"
+    !(pluginSettings().sastOnServerEnabled ?: false) -> " (disabled in Snyk.io)"
```

Comment:
are we ok with hardcoding `snyk.io` for potential(upcoming?) Single Tenant (onprem) version of Snyk engine?